src/Core/Abstractions/IAuthService.cs

```diff
@@ -32,7 +32,10 @@ public interface IAuthService
         Task<List<PasswordlessLoginResponse>> GetPasswordlessLoginRequestsAsync();
         Task<List<PasswordlessLoginResponse>> GetActivePasswordlessLoginRequestsAsync();
         Task<PasswordlessLoginResponse> GetPasswordlessLoginRequestByIdAsync(string id);
-        Task<PasswordlessLoginResponse> GetPasswordlessLoginResponseAsync(string id, string accessCode);
+        /// <summary>
+        /// This method doesn't required authentication
```

Comment:
⛏️ A summary should describe what the method does and then you can add additional stuff
```suggestion
        /// Gets a passwordless login request by <paramref name="id"/> and <paramref name="accessCode"/>. No authentication required.
```
We can leave it as "request" given that for now the signature of the method won't change.